schemas/Manual.locations.schema.json

```diff
@@ -92,6 +92,10 @@
                     "type": "boolean",
                     "default": false
                 },
+                "hint_entrance": {
```

Comment:
Should probably have a name that more clearly says what it's for. Since it's a location name that's just for hinting, maybe something like `hint_name`.

Reply:
> Considering it's shown under the "Entrance" column on the webtracker, I'm happy with the current name.

This also shows up in output from the `!hint` command in the client. 😃

Reply:
Even the `!hint` version maintains the `{location} in {world} at {entrance}` structure:

`[Hint]: CooltrainerTunic 's MP Berry x3 is at Shop - Coin 2 in AxxroyTunic 's World at Windmill Entrance . (not found)`